src/script/components/panel/UserDetails.tsx

```diff
@@ -92,13 +92,7 @@ const UserDetails: React.FC<UserDetailsProps> = ({
         </div>
       )}
 
-      {classifiedDomains && (
-        <ClassifiedBar
-          users={[participant]}
-          classifiedDomains={classifiedDomains}
-          style={{width: 'calc(100% + 32px)'}}
```

Comment:
Any reason it was added here? 🤔

Reply:
found the [ticket for ya](https://wearezeta.atlassian.net/browse/FS-386) and the [PR that came with it](https://github.com/wireapp/wire-webapp/pull/12552)

Reply:
Thanks, but I mean the style property haha 😄

Reply:
It kinda goes back to what we were discussing yesterday, that we don't really know if we should go with css-in-js or not, no?

Reply:
Please don't change this for now, this is only for federated environment. Maybe it can be added to the `css` property rather than `style` but this style is important (it a hack to have the right design in some occurences)